src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -274,27 +277,46 @@ extension RiskProvider: RiskProviding {
 			case .success(let _config):
 				appConfiguration = _config
 
-				self.determineSummary(
-					userInitiated: userInitiated,
-					ignoreCachedSummary: ignoreCachedSummary,
-					appConfiguration: _config,
-					completion: { [weak self] result in
-						guard let self = self else { return }
-
-						switch result {
-						case .success(let _summary):
-							summary = _summary
-						case .failure(let error):
-							Log.info("RiskProvider: Failed determining summary.", log: .riskDetection)
-							self.failOnTargetQueue(
-								error: error,
-								completion: completion
-							)
-						}
-
-						group.leave()
-					}
-				)
+                func determineSummaries() {
+                    self.determineSummary(
+                        userInitiated: userInitiated,
+                        ignoreCachedSummary: ignoreCachedSummary,
+                        appConfiguration: _config,
+                        completion: { [weak self] result in
+                            guard let self = self else { return }
+
+                            switch result {
+                            case .success(let _summary):
+                                summary = _summary
+                            case .failure(let error):
+                                Log.info("RiskProvider: Failed determining summary.", log: .riskDetection)
+                                self.failOnTargetQueue(
+                                    error: error,
+                                    completion: completion
+                                )
+                            }
+
+                            group.leave()
+                        }
+                    )
+                }
+
+                // The result of a hour package download is not handled, because for the risk detection it is irrelevant if it fails or not.
+                self.downloadHourPackages { [weak self] in
+                    guard let self = self else { return }
+
```

Comment:
Do we have logging if the downloading of hourly packages did fail? Might get useful for debuging.

Reply:
Nevermind just found it :D